src/_guides/language/effective-dart/usage.md

```diff
@@ -567,23 +568,21 @@ class Box {
   }
 }
 {% endprettify %}
-</div>
 
 Note that constructor parameters never shadow fields in constructor
 initialization lists:
 
-<div class="good">
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/usage_good.dart (param-dont-shadow-field-ctr-init)"?>
 {% prettify dart %}
 class Box extends BaseBox {
   var value;
 
   Box(value)
       : value = value,
-        super(value)
-      {}
+        super(value);
```

Comment:
I dropped `{}` so as to follow our own guidelines :) ... otherwise the analyzer (w/ lint rules enabled) complains.